features/src/main/feature/camel-features.xml

```diff
@@ -540,28 +540,28 @@
     <feature name='camel-azure-servicebus' version='${project.version}' start-level='50'>
         <feature version='${camel.osgi.version.range}'>camel-cloudevents</feature>
         <feature version='[1.46,1.47)'>azure</feature>
-        <bundle dependency='true'>wrap:mvn:com.azure/azure-core-amqp/2.9.1</bundle>
-        <bundle dependency='true'>wrap:mvn:com.azure/azure-messaging-servicebus/7.15.1</bundle>
+        <bundle dependency='true'>wrap:mvn:com.azure/azure-core-amqp/2.9.1$Bundle-Version=2.9.1</bundle>
+        <bundle dependency='true'>wrap:mvn:com.azure/azure-messaging-servicebus/7.15.1$Bundle-Version=7.15.1</bundle>
         <bundle>mvn:org.apache.camel.karaf/camel-azure-servicebus/${project.version}</bundle>
     </feature>
     <feature name='camel-azure-storage-blob' version='${project.version}' start-level='50'>
         <feature version='${camel.osgi.version.range}'>camel-cloudevents</feature>
         <feature version='[1.46,1.47)'>azure-storage</feature>
-        <bundle dependency='true'>wrap:mvn:com.azure/azure-storage-blob/12.25.2</bundle>
-        <bundle dependency='true'>wrap:mvn:com.azure/azure-storage-blob-changefeed/${azure-storage-blob-changefeed-version}</bundle>
+        <bundle dependency='true'>wrap:mvn:com.azure/azure-storage-blob/12.25.2$Bundle-Version=12.25.2</bundle>
+        <bundle dependency='true'>wrap:mvn:com.azure/azure-storage-blob-changefeed/${azure-storage-blob-changefeed-version}$Bundle-Version=12.0.0.beta_19</bundle>
```

Comment:
Why don't you use the maven property to set the version?

Reply:
The version coming from the version placeholder doesn't work
`<azure-storage-blob-changefeed-version>12.0.0-beta.19</azure-storage-blob-changefeed-version>`
```
karaf@root()> feature:install camel-azure-storage-blob
org.osgi.framework.BundleException: Could not create bundle object.
        at org.apache.felix.framework.Felix.installBundle(Felix.java:3339)
        at org.apache.felix.framework.BundleContextImpl.installBundle(BundleContextImpl.java:147)
        at org.eclipse.equinox.internal.region.BundleIdBasedRegion.installBundle0(BundleIdBasedRegion.java:120)
        at org.eclipse.equinox.internal.region.BundleIdBasedRegion.installBundleAtLocation(BundleIdBasedRegion.java:100)
        at org.apache.karaf.features.internal.service.BundleInstallSupportImpl.installBundle(BundleInstallSupportImpl.java:135)
        at org.apache.karaf.features.internal.service.FeaturesServiceImpl.installBundle(FeaturesServiceImpl.java:1145)
        at org.apache.karaf.features.internal.service.Deployer.deploy(Deployer.java:906)
        at org.apache.karaf.features.internal.service.FeaturesServiceImpl.doProvision(FeaturesServiceImpl.java:1069)
        at org.apache.karaf.features.internal.service.FeaturesServiceImpl.lambda$doProvisionInThread$13(FeaturesServiceImpl.java:1004)
        at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
        at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
        at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
        at java.base/java.lang.Thread.run(Thread.java:833)
Caused by: java.lang.IllegalArgumentException: invalid version "12.0.0-beta.19": non-numeric "0-beta"
        at org.osgi.framework.Version.parseInt(Version.java:169)
        at org.osgi.framework.Version.<init>(Version.java:134)
        at org.osgi.framework.Version.valueOf(Version.java:255)
        at org.osgi.framework.Version.parseVersion(Version.java:226)
        at org.apache.felix.framework.util.manifestparser.ManifestParser.<init>(ManifestParser.java:129)
        at org.apache.felix.framework.BundleRevisionImpl.<init>(BundleRevisionImpl.java:117)
        at org.apache.felix.framework.BundleImpl.createRevision(BundleImpl.java:1290)
        at org.apache.felix.framework.BundleImpl.<init>(BundleImpl.java:114)
        at org.apache.felix.framework.Felix.installBundle(Felix.java:3287)
        ... 12 more
Caused by: java.lang.NumberFormatException: For input string: "0-beta"
        at java.base/java.lang.NumberFormatException.forInputString(NumberFormatException.java:67)
        at java.base/java.lang.Integer.parseInt(Integer.java:668)
        at java.base/java.lang.Integer.parseInt(Integer.java:786)
        at org.osgi.framework.Version.parseInt(Version.java:167)
        ... 20 more
```